extensions/amp-story/1.0/amp-story.js

```diff
@@ -1835,6 +1856,32 @@ export class AmpStory extends AMP.BaseElement {
         containsMediaElementWithAudio || hasStoryAudio);
   }
 
+  /**
+   * Checks for the presence of a sidebar. If a sidebar does exist, then an icon
+   * permitting for the opening/closing of the sidebar is shown.
+   * @private
+   */
+  initializeSidebar_() {
+    this.sidebar_ = this.element.querySelector('amp-sidebar');
+    if (this.sidebar_) {
```

Comment:
Nit: to remove one level of indentation for the following block of code, could you do `if (!this.sidebar_) return;` instead?